src/guest/vm_builder_qemu.cc

```diff
@@ -222,12 +229,20 @@ static int SetAvailableVf(void) {
         return false;
     }
 
-    for (int i = 1; i < totalvfs; i++) {
+    for (int i = 1; i < current_vfs; i++) {
         std::string sd(kIntelGpuDevPath);
         sd.replace(sd.end() - 2, sd.end(), std::to_string(i) + "/enable");
         int status = ReadSysFile(sd.c_str(), std::ios_base::dec);
-        if (status == 0)
+        if (status == 0) {
+            std::string timeoutPath = kTimeoutPath;
+            timeoutPath.append(std::to_string(i) + kTimeoutPremptFile);
+            WriteSysFile(timeoutPath.c_str(), std::to_string(kPreintTimeout));
+
+            std::string execQuantumPath = kTimeoutPath;
+            execQuantumPath.append(std::to_string(i) + kTimeoutExecQuantum);
+            WriteSysFile(execQuantumPath.c_str(), std::to_string(kExecQuantum));
```

Comment:
Could you add more comments in commit message about why add this and how it works?

Reply:
These are provided by SRIOV team, to configure the GPU firmware. Preemption Timeout handles the timeout before engine reset. Execution Quantum defines the max time slice.

Please refer: https://wiki.ith.intel.com/display/hspesws/Guc+Tunables

Reply:
Please add the summerized info in commit message(exclude internal link).